modules/serverless-overriding-pdbs-eventing.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * /serverless/eventing/tuning/serverless-ha.adoc
+
+
+:_content-type: PROCEDURE
+[id="serverless-overriding-poddistruptionbudgets-eventing_{context}"]
+= Overriding disruption budgets
+
+A Pod Disruption Budget (PDB) is a standard feature of Kubernetes APIs that helps limit the disruption to an application when its pods need to be rescheduled for maintenance reasons.
```

Comment:
I might move the sentence and note before the .Procedure heading, as [we aren't supposed to have sentences after that heading](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs) (I think).
Also, the procedure needs one or more steps (if only one, use an asterisk).